api-controllers/FabrikBaseController.js

```diff
@@ -127,6 +127,12 @@ class FabrikBaseController extends BaseController {
   }
 
   validateRestoreQuota(options) {
+    const isPitrEnabled = this.getService(options.service_id).pitr;
+    if (!isPitrEnabled) {
+      // For non pitr supported services skipping quota check
```

Comment:
remove comments which are self-explanatory from looking at the code directly & straight forward.

Reply:
done